task/cerberus-check/0.1/cerberus-check.yaml

```diff
@@ -0,0 +1,76 @@
+---
+kind: Task
+apiVersion: tekton.dev/v1beta1
+metadata:
+  name: cerberus-check
+  labels:
+    app.kubernetes.io/version: "0.1"
+    app.kubernetes.io/name: "cerberus-check-task"
+    tool: cerberus
+  annotations:
+    tekton.dev/categories: Testing
+    tekton.dev/displayName: cerberus-check
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/platforms: linux/amd64
+    tekton.dev/tags: cerberus, chaos, test
+spec:
+  description: >-
+    The cerberus-check `task`` execute periodic check of a cerberus server and return
+    a OK when cerberus signal is positive.
+
+    Cerberus is a cluster watchdog who return health status of openshift or kubernetes
+    cluster. This task allow you to check this signal as part of your chaos pipeline.
+  params:
+    - name: cerberus_server
+      type: string
+      description: Full URL to the cerberus server
+      default: "http://cerberus:8080"
+    - name: timeout
+      type: string
+      description: Timeout for the healthcheck performed against the cerberus server
+      default: "60"
+    - name: frequency
+      type: string
+      description: Frequency of the healthcheck in case of bad pulse
+      default: "10"
+  steps:
+    - image: "quay.io/startx/apache:alpine3"
```

Comment:
can you please parameterise the image so that if someone wants to use their own then they can use it?
*not urgent/important*:
I can see that the image used here is being built by you, can you please make it multi-arch so that it can be used on different clusters as well?

Reply:
resolved in commit https://github.com/tektoncd/catalog/pull/1004/commits/0af896eed897652ed1037fffea3704b03b1c7b5c